### PR TITLE
Update fing.rb 3.0 download URL and hash.

### PR DESCRIPTION
**Pull request by geerlingguy**
### Checklist
- [x] The commit message includes the cask’s name and version.
- [x] `brew cask audit --download fing.rb` is error-free.
- [x] `brew cask style --fix fing.rb` left no offenses.

### CODE REVIEW

File: Casks/fing.rb [hunk 1]
```diff
@@ -1,10 +1,11 @@
 cask 'fing' do
   version '3.0'
-  sha256 '207c6f441c8edb55b4a59b57f0d5e487afcf578a637d0981e2027adb4f848ea1'
+  sha256 'a8497ce00d58609d8677c6c7850e479420516d94e9f37d681dbdc970359294c6'
 
-  url 'https://www.fingbox.com/download?plat=osx'
+  # wpengine.netdna-ssl.com was verified as official when first introduced to the cask
```
Comment (vitorgalvao) [inline Casks/fing.rb]: Needs to be more specific (`39qiv73eht2y1az3q51pykkf-wpengine.netdna-ssl.com`).

File: Casks/fing.rb [hunk 2]
```diff
@@ -1,10 +1,11 @@
...
+  url 'https://39qiv73eht2y1az3q51pykkf-wpengine.netdna-ssl.com/wp-content/uploads/2016/09/overlook-fing-3.0.dmg_.zip'
```
Comment (vitorgalvao) [inline Casks/fing.rb]: `url "https://39qiv73eht2y1az3q51pykkf-wpengine.netdna-ssl.com/wp-content/uploads/#{version.after_comma}/overlook-fing-#{version.before_comma}.dmg_.zip"`

File: Casks/fing.rb [hunk 3]
```diff
@@ -1,10 +1,11 @@
 cask 'fing' do
   version '3.0'
```
Comment (vitorgalvao) [inline Casks/fing.rb]: `version '3.0,2016/09'`

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

File: Casks/fing.rb (hunk 1 above)
Comment (reitermarkus) [inline Casks/fing.rb]: This is too unspecific, use `39qiv73eht2y1az3q51pykkf-wpengine.netdna-ssl.com`.

**Review by reitermarkus — CHANGES_REQUESTED** [review]

File: Casks/fing.rb (hunk 3 above)
Comment (geerlingguy) [inline Casks/fing.rb]: Resolved in latest revision.

File: Casks/fing.rb (hunk 1 above)
Comment (geerlingguy) [inline Casks/fing.rb]: Resolved in latest revision.

File: Casks/fing.rb (hunk 2 above)
Comment (geerlingguy) [inline Casks/fing.rb]: Resolved in latest revision.

File: Casks/fing.rb (hunk 1 above)
Comment (geerlingguy) [inline Casks/fing.rb]: Resolved in latest revision.

**Comment by geerlingguy** [comment]
@vitorgalvao @reitermarkus - Thanks for the review! I made the changes as requested.

**Review by vitorgalvao — APPROVED** [review]

**Comment by vitorgalvao** [comment]
Thank you.

**Pull request merged**